e2e/features/animation/mobile-animation-test.js

```diff
@@ -0,0 +1,44 @@
+const localQueryStrings = require('../../reuseables/querystrings.js');
+
+const TIME_LIMIT = 10000;
+
+module.exports = {
+  before: (c) => {
+    c.url(c.globals.url + localQueryStrings.knownDate);
+    c.pause(10000);
+    c.setWindowSize(375, 667);
+    c.pause(10000);
+  },
+
+  'Mobile animate button opens widget': (c) => {
+    c.waitForElementVisible('.mobile-animate-button', TIME_LIMIT);
+    c.useCss().click('.mobile-animate-button');
+    c.pause(500);
+    c.waitForElementPresent('#wv-animation-widget', TIME_LIMIT);
+    c.expect.element('.custom-interval-delta-input').to.have.value.that.equals('1');
+    c.expect.element('.dropdown-toggle').text.to.equal('DAY');
+  },
+
+  'Minimizing mobile animation widget opens collapsed animation widget': (c) => {
+    c.useCss().click('.wv-minimize');
+    c.waitForElementVisible('#collapsed-animate-widget-phone-portrait', TIME_LIMIT);
+  },
+
+  'Playing the animation changes the date of the mobile date picker': (c) => {
+    c.useCss().click('#collapsed-animate-widget-phone-portrait');
+    c.pause(3000);
```

Comment:
I was able to apply all of the recommended changes from this thread but I did leave this one instance of a long pause. This one is because I need to wait for the animation to load and play and I'm trying to account for poor connections when it may take longer to load despite the fact that it is a smaller animation.